_posts/2020-11-09-ten-tips-for-tidy-data.markdown

```diff
@@ -0,0 +1,66 @@
+
+This seems like a very simple concept, but there are some very common mistakes that violate this structure.
+
+One very common problem is using column headers that are values instead of the name of a variable. In the example below, each cell contains an observation, but the column header, the date of the observation, is in fact another observation, that should have its own column.
+
+![column header as values]({{ site.url }}/assets/table_4_column_headers_with_values.png)
+
+It should be re-organized so that the date is in its own column, and the column headers are descriptive, but don’t store any data themselves.
+
+![column headers tidied]({{ site.url }}/assets/table_6_tidy_columns.png)
+
+
+## 2.	Be consistent.
+It is very important to be consistent. Whatever you do, keep it consistent.  This is especially important when working with collaborators, but it also helps you with comparisons and analysis, and is increasingly important the larger your dataset. This rule is crucial to preventing errors. Choose a format and stick to it throughout your dataset. It may help to have comments or a data dictionary to remind yourself of your format. Examples of being consistent include:
+
+* Use consistent variable names (“leaf_number” vs. “num_of_leaves”)
+* Use consistent codes for categorical variables (“female” vs. “F”)
+* Use a consistent code for any missing values (“NA” vs. “—” vs. a blank cell)
+
+## 3.	Date format
+Dates should be formatted using the global ISO 8601 standard which is YYYY-MM-DD. For example, 2020-11-06 is November 6th, 2020. This is standard and helps to avoid confusion between collaborators of different countries. Beware the automatic date formatting that happens in excel!
+
+## 4.	Choose good names
+In general, think carefully about how you name things. This applies to both file names and column names. They should be descriptive enough to avoid confusion, but concise enough to be functional. As a general rule, do not use spaces or other special characters in file or column names, except for underscores or hyphens. And of course, think about Tips #2 and #3 when naming things.
+
+## 5.	No empty cells
+It is generally accepted to use “NA” when a cell is meant to contain no value. This makes it clear that the cell was not left blank by mistake. And this will come in handy later, when sorting or manipulating data, to know the common code for a missing value.
+
+## 6.	Just one value per cell
+During data collection, there may be reason to store multiple, related data values in the same cell. Perhaps a code identifying an individual, that contains two pieces of information about that individual (like “adult_male_01”) or two related measurements (like “length x width”). But, in a tidy spreadsheet, these should be treated like two distinct pieces of data, and split into different columns accordingly. In the future, you may want to sort or compare your data by these different variables, and you’ll be glad you split them.
```

Comment:
"treated like two distinct pieces of data, and split into different columns accordingly" no comma is needed

Reply:
Ok, comma removed.